infrastructure/modules/cluster/dashboard.tf

```diff
@@ -0,0 +1,244 @@
+resource "azurerm_dashboard" "doc-index-updater-dashbaord" {
+  name                = "doc-index-updater-dashboard"
+  resource_group_name = var.resource_group_name
+  location            = var.location
+
+  tags = {
+    Environment = var.environment
+  }
+
+  dashboard_properties = <<DASH
+{
+  "lenses": {
+    "0": {
+      "order": 0,
+      "parts": {
+        "0": {
+          "position": {
+            "x": 0,
+            "y": 0,
+            "colSpan": 6,
+            "rowSpan": 4
+          },
+          "metadata": {
+            "inputs": [
+              {
+                "name": "ComponentId",
+                "value": {
+                  "SubscriptionId": "bec11470-1346-4cdd-af2e-ce1f360671a1",
```

Comment:
is this a secret?

Reply:
You can't actually do anything with a subscription ID on it's own so I would say no, but I would agree that it's preferable to keep it private. This is should all be parameterised now so prod subscription ID will never be made public.

[More discussion](https://stackoverflow.com/a/45663968)